src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -1100,21 +1114,100 @@ subroutine decomp_rates_mimics(bounds, num_soilc, filter_soilc, &
       mimics_cn_r = params_inst%mimics_cn_r
       mimics_cn_k = params_inst%mimics_cn_k
 
+      ! Local column-level ligninNratioAvg because
+      ! not available when use_fates = .true.
+      ! Also get FATES copy of annsum_npp for FATES cases
+      if (use_fates) then
+         ! Initialize
+         do fc = 1,num_soilc
+            c = filter_soilc(fc)
+            ligninNratioAvg_local(c) = 0._r8
+            annsum_npp_col_local(c) = 0._r8
+         end do
+
+         ! Loop over p to get FATES copy of annsum_npp and local ligninNratio,
+         ! both for use_fates = .true.
+         nc = bounds%clump_index
+         do fp = 1, num_soilp
+
+            p = filter_soilp(fp)
+            c = patch%column(p)
+
+            pf = p - col%patchi(c)
+            s  = clm_fates%f2hmap(nc)%hsites(c)
+            annsum_npp(p) = clm_fates%fates(nc)%bc_out(s)%annsum_npp_pa(pf)
+
+            ! Values by pft from Table 1 in Brovkin et al. (2012):
+            ! https://bg.copernicus.org/articles/9/565/2012/bg-9-565-2012.pdf
+            ! Not moving to the params files at this time: Hardwiring will
+            ! become unnecessary when we calc. ligninNratioAvg the same way for
+            ! FATES and non FATES cases.
+            select case (pftname(patch%itype(p)))
```

Comment:
If use_fates is true, then none of these patch level PFTs are actually valid.  I need to read through this code more to figure out how non-fates mimics arrives at this ligninRatio to give any recommendations thought.  brb

Reply:
@slevisconsulting and @wwieder , I see two ways to handle this:
1) just use a single representative constant for any and all FATES patches.
2) we could have FATES pass in this ratio right now. Even though we don't have the dynamic FATES N model coupled with CLM yet, we do have the stoichiometries and the lignin fractions of all of our fates generated litter.

Perhaps we should use item 1 for right now, and we can have a follow up PR to add in item 2? (I could help with that one)

Reply:
This plan seems reasonable.  @slevisconsulting how do we keep track of this suggestion # 2 from @rgknox? 

Alternatively, if it's not too difficult maybe we just do this right from the start?

Reply:
@rgknox let's follow up to pursue option 2. How's your availability this wk on Weds, Thurs, or Friday afternoons?

Reply:
In our meeting @rgknox and I agreed that he would make the necessary changes on the fates side to accomplish item 2 above. Timeframe about 2-4 weeks.